app/views/home/jobs.html.erb

```diff
@@ -6,18 +6,7 @@
 	          <h2>Join our team!</h2>
 	          <hr>
         </div>
-        <p class="large">We are looking for enthusiastic individuals to join our team. Check out our open positions below!</p>
```

Comment:
Curious if we could opt for commenting this out for future use when we do have jobs again? Just wanting to keep it like an on/off switch so that anyone can dive in and see what the original style was...what do you think?